index.html

```diff
@@ -39,34 +39,34 @@ <h1>BSDLabs</h1>
         </header>
         <section id="main_content">
             <p>
-                FreeBSD Labs is a dedicated online space and initiative geared
+                FreeBSD Labs is an online space and initiative geared
                 towards the realisation of community-driven THINK BIG and
                 DIFFERENT ideas.
             </p>
             <p>
-                We want to brainstorm and prototype next-gen concepts that:
+                We brainstorm and prototype next-gen concepts that:
             </p>
             <ul>
-                <li>Can begin with a Prototype or MVP, then scale from there</li>
-                <li>Can be achieved with a small group (2-4) of diversely skilled people</li>
-                <li>Are not just related to “code” (src, ports, doc)</li>
-                <li>Are not just “feature requests”</li>
-                <li>Someone (YOU!) are willing to drive forward</li>
+                <li>Can begin with a prototype or <a href="https://en.wikipedia.org/wiki/Minimum_viable_product">MVP</a>, then scale from there</li>
+                <li>Can be achieved with a small group (2–4) of diversely-skilled people</li>
+                <li>Are not just related to <em>code</em> (src, ports, doc)</li>
+                <li>Are not just feature requests</li>
+                <li>Someone (YOU!) is willing to drive forward</li>
             </ul>
             <p>
-                Think <a href="https://en.wikipedia.org/wiki/Skunkworks_project" target="_blank">“Skunk Works”</a> for FreeBSD. Welcome to The Lab!
+                Think <a href="https://en.wikipedia.org/wiki/Skunkworks_project" target="_blank"><em>skunkworks</em></a> for FreeBSD. Welcome to the Labs!
             </p>
 
             <h2>Getting started</h2>
             <ul>
                 <li>Join us on <a href="https://github.com/bsdlabs" target="_blank">GitHub</a></li>
-                <li>Stop by #bsdlabs @ Libera Chat or our Discord (jump on IRC for an invite)</li>
+                <li>Stop by #bsdlabs @ Libera Chat or our Discord (for an invite: jump on IRC, or into the <a href="https://discord.com/channels/727023752348434432/757543661058654269">#community</a> channel in Discord for FreeBSD)</li>
```

Comment:
* Link Discord to `https://discord.gg/9vZzzxz7y9` directly (i've done this elsewhere)
* Remove freebsd discord (#community) link, not relevent (globally, yet)
* Re-order Discord before IRC